README.md

```diff
@@ -67,7 +66,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Install Java
-        uses: useblacksmith/setup-java@v5
```

Comment:
This should be `useblacksmith/setup-java@v5` for our custom runners

Reply:
My thought process was, that the average user would be more familiar with the actions/setup-java@v4 you see everywhere. We could mention blacksmith as a faster alternative, but the README should show the fastest way to this up.

Reply:
ohhh I just spotted this was in the readme